common/lib/client/resource.ts

```diff
@@ -51,16 +45,19 @@ function unenvelope(callback: Function, format: Utils.Format | null) {
       }
     }
 
-    if (body.statusCode === undefined) {
+    if (!body) {
+      callback(new ErrorInfo('Body is missing', null));
```

Comment:
Would be good to add something to this error message to indicate where it's coming from, eg "unenvelope(): Body is missing"

Reply:
Done